### PR TITLE
External content, fedora 5.0, and triplestore indexing fixes

### PR DESCRIPTION
**Pull request by dannylamb**
Part of https://github.com/Islandora-CLAW/CLAW/issues/1079

Deploys alpaca with new features repos to handle the new fcrepo5 client, fcrepo-camel-toolbox, camel, and activemq.

Test with https://github.com/Islandora-Devops/claw-playbook/pull/100

### CODE REVIEW

**Comment by whikloj** [comment]
@dannylamb are you doing stuff here to switch back to the deployed artifacts or is it good? 

I'm wondering mostly about [this line](https://github.com/Islandora-Devops/ansible-role-alpaca/pull/3/files#diff-7eeda618087b49ae876084ab6c73fdbbR10)

File: defaults/main.yml [hunk 1]
```diff
@@ -1,14 +1,18 @@
 ---
 
 alpaca_from_source: no
```
Comment (whikloj) [inline defaults/main.yml:3]: If this PR still needs to go in, should this line be removed as it appears that you removed the parts from `tasks/deploy.yml` that refer to it?

And if it goes I guess the entire `tasks/compile.yml` could go to?

Comment (dannylamb) [reply defaults/main.yml:3]: I'll remove what's duplicated in the playbook and let the role define these defaults. The playbook also can now point to the latest build and need not compile from source, so there's more for me to remove once this gets merged.

Comment (dannylamb) [reply defaults/main.yml:3]: Now that I think about it, let me add more from the playbook to this PR.

Comment (dannylamb) [reply defaults/main.yml:3]: @whikloj @seth-shaw-unlv Good to go now. Once this goes in I'll remove the duplicates in https://github.com/Islandora-Devops/claw-playbook/pull/100

File: defaults/main.yml [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-
 alpaca_version: master
 alpaca_clone_directory: /opt/alpaca
 
 alpaca_karaf_repos:
+  - mvn:org.apache.camel.karaf/apache-camel/2.20.4/xml/features
+  - mvn:org.apache.activemq/activemq-karaf/5.15.0/xml/features
   - mvn:ca.islandora.alpaca/islandora-karaf/LATEST/xml/features
```
Comment (dannylamb) [inline defaults/main.yml]: Do we want to stop now and start using tagged versions for our feature?  @ajstanley got bit by that one this morning and spun up a new playbook only to receive the new version of alpaca but the old version of everything else.

Comment (seth-shaw-unlv) [reply defaults/main.yml]: Probably...

**Review by seth-shaw-unlv — APPROVED** [review]
Here we go!

**Pull request merged**